pom.xml

```diff
@@ -113,6 +113,11 @@
             <artifactId>dropwizard-swagger</artifactId>
             <version>2.0.0-1</version>
         </dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
```

Comment:
do you need this dependency?

Reply:
Yes, was throwing an error when trying to build the api within the image